### PR TITLE
[Static reference] LPS-181106 layout-content-page-editor-web, layout-admin-web, layout-taglib

### PR DESCRIPTION
**Pull request by JanisZhang**
previous: https://github.com/liferay-core-infra/liferay-portal/pull/4184
https://issues.liferay.com/browse/LPS-181106

changes in this pull:
- inline AssetHelperUtil as it's only used by AssetListEntryUsagesManager
- put InfoSearchClassMapperRegistryUtil where InfoSearchClassMapperRegistry is.

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/util/AssetListEntryUsagesManager.java [hunk 1]
```diff
@@ -84,6 +113,57 @@
 @Component(service = AssetListEntryUsagesManager.class)
 public class AssetListEntryUsagesManager {
 
+	public List<AssetPublisherAddItemHolder> getAssetPublisherAddItemHolders(
```
Comment (dantewang) [inline modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/util/AssetListEntryUsagesManager.java:116]: @JanisZhang This method doesn't need to be public?

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:echo

**Pull request closed without merging**